earn/src/data/LendingPair.ts

```diff
@@ -360,6 +362,12 @@ export async function getLendingPairBalances(
   const deprecatedLendingPairBalancesArray: LendingPairBalances[] = [];
   const balancesMap: LendingPairBalancesMap = new Map();
 
+  balancesMap.set('0x0000000000000000000000000000000000000000', {
```

Comment:
Could we make this (empty address) a constant to avoid silly mistakes?

Reply:
Turns out we already had one :)